pkgs/development/libraries/libaom/default.nix

```diff
@@ -13,6 +13,32 @@ stdenv.mkDerivation rec {
   buildInputs = [ perl yasm ];
   nativeBuildInputs = [ cmake pkgconfig python3Packages.python ];
 
+  cmakeFlags = [ "-DBUILD_SHARED_LIBS=ON" ];
+
+  # * libaom tries to detect what version it is
+  #   * I couldn't get it to grab this from git info,
+  #     (which needs at least leaveDotGit=true)
+  #   * it uses a perl script to parse from CHANGELOG,
+  #     but 1.0.0 doesn't contain an entry for itself :(
+  # * Upstream patch that adds the entry also nukes all of
+  #   the versions before a project change (open-sourcing?)
+  #   and as a result is 34K which is way too big just to fix this!
+  #   * A stable URL to fetch this from works, but...
+  # * Upon inspection the resulting CHANGELOG is shorter
+  #   than this comment, so while yes this is a bit gross
+  #   adding these 4 lines here does the job without
+  #   a huge patch in spirit of preferring upstream's fix
+  #   instead of `sed -i 's/v0\.1\.0/v1.0.0/g' aom.pc` or so.
```

Comment:
There build is pretty much over-engineered.
As simple ./create-relase 1.x that replaces a few strings would have also done the job.